themes/docsy/assets/scss/_kiali-home.scss

```diff
@@ -321,6 +321,8 @@ section#blog-latest div#blog-blocks > a > div p {
   section#blog-latest div#blog-blocks {
```

Comment:
Is this scss actually used?

Reply:
I don't know. Probably not.
I notice it was used in the blog cards. But since I removed it, I'm not sure if it is being used somewhere else. Surely, it is not used in the landing page.